docs/source/testing/mocking.md

```diff
@@ -7,26 +7,30 @@ Mocking enables Apollo Server to return simulated data for GraphQL operations ba
 
 Mocking enables frontend developers to build out and test UI components and features without needing to wait for a full backend implementation. Mocking is also valuable when using a UI tool like [Storybook](https://storybook.js.org/), because you don't need to start a real GraphQL server.
 
-## Using default mocks
+## Enabling mocks
 
-You can turn on Apollo Server's default mocking logic by passing `mocks: true` to the ApolloServer constructor:
+To mock data based on your schema start by installing the [`@graphql-tools/mock`](https://www.graphql-tools.com/docs/mocking) and [`@graphql-tools/schema`](https://www.graphql-tools.com/docs/generate-schema#makeexecutableschemaoptions) packages into your dev dependencies:
 
-```js{11}
-const { ApolloServer, gql } = require('apollo-server');
+```bash
+  npm install --save-dev @graphql-tools/mock @graphql-tools/schema
+```
 
-const typeDefs = gql`
-  type Query {
-    hello: String
-  }
-`;
+You can combine `addMocksToSchema` (from `@graphql-tools/mock`) and `makeExecutableSchema` (from `@graphql-tools/schema`) to provide mock data for every field in your server's schema:
 
-const server = new ApolloServer({
-  typeDefs,
-  mocks: true,
-});
+```ts
+import { ApolloServer } from '@apollo/server';
+import { addMocksToSchema } from '@graphql-tools/mock';
+import { makeExecutableSchema } from '@graphql-tools/schema';
+import { typeDefs } from './schema';
+import { resolvers } from './resolvers';
 
-server.listen().then(({ url }) => {
-  console.log(`🚀 Server ready at ${url}`)
+new ApolloServer({
+  // addMocksToSchema accepts a schema instance to
```

Comment:
to what?

Reply:
Hahaha, thank you! I'm not sure how that line got caught off, but I've decided to rephrase it (minus the highlights):

```
new ApolloServer({
  // addMocksToSchema accepts a schema instance and provides
  // mocked data for each field in the schema
  schema: addMocksToSchema({
    schema: makeExecutableSchema({ typeDefs, resolvers }),
  }),
});

```